provision.sh

```diff
@@ -95,4 +109,4 @@ su -c "/usr/bin/katana --update"
 
 katana install katana
 katana start katana
-systemctl enable samurai-katana
```

Comment:
Not sure where this line came from, and if it should be different. I didn't write it..

Reply:
Eh, I'm sure it's fine.